File: URI/1074.c
#include<stdio.h>
#include<stdlib.h>

int N,n;

int main()
{
    scanf("%d",&N);
    for(N;N>0;N--)
    {
        scanf("%d",&n);
        if(n%2==0)
        {
            if(n!=0)
            {
                printf("EVEN ");
            }
            else
            {
                printf("NULL\n");
            }
        }
        else
        {
            printf("ODD ");
        }
        if(n>0)
        {
            printf("POSITIVE\n");
        }
        if(n<0)
        {
            printf("NEGATIVE\n");
        }
    }
    return 0;
}

File: URI/Python/1074.py
N = int(input())

i = 0
while i < N:
	valor = int(input())
	if valor == 0:
		print("NULL")
	elif valor % 2 == 0:
		print("EVEN", end = " ")
	elif valor % 2 == 1:
		print("ODD", end = " ")

	if valor > 0:
		print("POSITIVE")
	elif valor < 0:
		print("NEGATIVE")
	i += 1